runtime/lua/vim/lsp/inlay_hint.lua

```diff
@@ -402,14 +403,20 @@ end
 --- Buffer number, or 0 for current buffer, or nil for all.
 --- @field bufnr integer?
 
---- Enables or disables inlay hints for a buffer.
+--- Enables or disables inlay hints for the |filter|ed scope.
```

Comment:
What does `|filter|ed` scope precisely mean? The help entry `|filter|` doesn't seem relevant.

Reply:
Oh yes, it's a misuse exactly. Would the new way of writing this be better?